packages/table/src/common/classes.ts

```diff
@@ -44,6 +45,8 @@ export const TABLE_OVERLAY_LAYER = "bp-table-overlay-layer";
 export const TABLE_POPOVER_WHITESPACE_NORMAL = "bp-table-popover-whitespace-normal";
 export const TABLE_POPOVER_WHITESPACE_PRE = "bp-table-popover-whitespace-pre";
 export const TABLE_REGION = "bp-table-region";
+export const TABLE_REORDERING = "bp-table-reordering";
+export const TABLE_REORDERING_CURSOR_SHIELD = "bp-table-reordering-cursor-shield";
```

Comment:
does this class need to be here? seems unlikely anyone would want to access this one.